kedro_kubeflow/generators/utils.py

```diff
@@ -44,3 +38,23 @@ def create_container_environment():
             env_vars.append(k8s.V1EnvVar(name=key, value=os.environ[key]))
 
     return env_vars
+
+
+def create_command_using_params_dumper(command):
```

Comment:
This may look scary, but it's actually quite simple:

1. Kedro params land now as container arguments in format `['key1', 'value1', 'key2', 'value2', ...]`
2. Container executes bash, bash intercepts the arguments
3. The arguments are passed to `python` call using `"$@"`
4. Python call pushes these into `config.yaml` file

In this case there is no need to modify anything in the project's kedro cli file